mediapipe/tasks/ios/text/text_classifier/sources/MPPTextClassifier.h

```diff
@@ -0,0 +1,102 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+#import "mediapipe/tasks/ios/core/sources/MPPTaskOptions.h"
+#import "mediapipe/tasks/ios/text/text_classifier/sources/MPPTextClassifierOptions.h"
+#import "mediapipe/tasks/ios/text/text_classifier/sources/MPPTextClassifierResult.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * This API expects a TFLite model with (optional) [TFLite Model
```

Comment:
Can you add "Performs classification on text." before (as an introduction to what this dows).